From 236be31c99a6da0ed14d61f3e553d9114ae2743e Mon Sep 17 00:00:00 2001
From: matt rice <ratmice@gmail.com>
Date: Sat, 30 Oct 2021 10:04:54 -0700
Subject: [PATCH] user-agent: use crate::USER_AGENT, Unsure where the other
 constant came from (reqwest maybe?). This could use more looking into other
 places this constant is used.

---
 podcasts-data/src/downloader.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/podcasts-data/src/downloader.rs b/podcasts-data/src/downloader.rs
index 20310316..3cc1c440 100644
--- a/podcasts-data/src/downloader.rs
+++ b/podcasts-data/src/downloader.rs
@@ -77,7 +77,7 @@ fn download_into(
     let client = reqwest::blocking::Client::builder()
         .redirect(policy)
         .referer(false)
-        .user_agent(USER_AGENT)
+        .user_agent(crate::USER_AGENT)
         .build()?;
     let mut resp = client.get(url).send()?;
     info!("Status Resp: {}", resp.status());
-- 
GitLab

